tools/repo/notify.py

```diff
@@ -227,6 +233,12 @@ def is_contrib(self, pr):
                 return True
         return False
 
+    async def is_reviewed(self, pr):
+        async for review in self.repo.getiter(f"pulls/{pr['number']}/reviews"):
+            if review["state"] == "APPROVED" or "/lgtm" in review["body"]:
```

Comment:
im not too sure about this - it doesnt check who is doing the reviewing at all or why they lgtm - eg it may have only been for api/deps/coverage/etc

how about a slightly more explicit approach - lets add a `review:needs-maintainer` label

we would then need to add a label setter in repokitteh or gh actions to allow people to set it - but we could eg check they are in reviewers.yaml or a relevant group

Reply:
I don't think we'd need to check for permission - this change is only making the nag happen less when it's not necessary. Allowing random strangers to pass by and change the state to needs-maintainer would *in the worst case* just give us the behavior we already have before this change.